lbz/events/api.py

```diff
@@ -0,0 +1,58 @@
+import json
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
```

Comment:
Put such `if` conditions below all the normal imports, otherwise they are wrongly sorted 😉